share/site/duckduckgo/base.tx

```diff
@@ -9,7 +9,7 @@
   <head>
     <: include "head.tx" :>
   </head>
-  <body id="pg-<: $f.filebase :>" class="page-<: $f.filebase :><: if $homepage { :> body--home<: } :>">
```

Comment:
Oh wow I think what happened was that I pushed into an existing branch. Not sure if that branch existed on my machine or remotely. 

Will pick off those commits.